src/nvim/fileio.c

```diff
@@ -2792,8 +2784,8 @@ buf_write (
         /*
          * Isolate one directory name, using an entry in 'bdir'.
          */
-        (void)copy_option_part(&dirp, copybuf, BUFSIZE, ",");
-        rootname = get_file_in_dir(fname, copybuf);
+        (void)copy_option_part(&dirp, IObuff, IOSIZE, ",");
```

Comment:
Curious about why you removed `copybuf`. Is it because the code removed below means `IObuff` is not used for other steps? Might be less risky to keep `copybuf`, otherwise we need to make sure `IObuff` is not used by any other codepaths in this area.

Reply:
As far as I understand `copybuf` was being used for the copy operation which is now taken care of by libuv. That's why I removed it. It was, however, being used as an argument to `copy_option_part`, but then I found `copy_option_part` being called this way at https://github.com/Said6289/neovim/blob/master/src/nvim/fileio.c#L2911